### ISSUE TITLE
[calendar] enhance current print view of agenda list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#280]
```
See attachment


```

Original issue reported on code.google.com by `rol...@liebl.ath.cx` on 3 Jan 2012 at 1:37

Attachments:
- [druckansicht_spalten.pdf](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-280/comment-0/druckansicht_spalten.pdf)

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 17 Feb 2012 at 7:49
- Changed title: **[Calendar] add new print view**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 17 Feb 2012 at 7:53
- Changed title: **[calendar] add new print view**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 21 Mar 2012 at 3:52
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 3 Apr 2012 at 4:31
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 3 Apr 2012 at 4:32
- Changed title: **[calendar] enhance current print view of agenda list**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in calendar v8.2,

You have to install plugin_manager to fetch recent calendar version.
```

Original comment by `rol...@liebl.ath.cx` on 4 Apr 2012 at 8:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 4 Apr 2012 at 8:57
- Changed state: **Fixed**
